apigateway_template.yaml

```diff
@@ -25,7 +25,7 @@ Resources:
   TestApiGateway:
     Type: AWS::Serverless::Api
     Properties:
-      Name: "axelra-interview-challenge-api"
+      Name: "axelrachallenge"
```

Comment:
Is there a reason for this renaming? bug because of dashes?

Reply:
No reason.